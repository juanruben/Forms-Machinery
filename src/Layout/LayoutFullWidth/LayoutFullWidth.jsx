import React from 'react';
import './LayoutFullWidth.scss';

const LayoutFullWidth = ({ children }) => (
    <div className="full-width-container">
        {children}
    </div>
);

export default LayoutFullWidth;
