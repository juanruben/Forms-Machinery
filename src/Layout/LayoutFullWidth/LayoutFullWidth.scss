.full-width-container{
	align-items: center;
	background-color: white;
	display: flex;
	justify-content: center;
	flex-direction: column;
	height: 100vh;
	width: 100%;
}