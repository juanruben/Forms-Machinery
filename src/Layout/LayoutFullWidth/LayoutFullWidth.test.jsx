import React from 'react';
import { shallow } from 'enzyme';
import LayoutFullWidth from './LayoutFullWidth';

describe('LayoutFullWidth', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<LayoutFullWidth />);
    });

    it('should render correctly', () => expect(wrapper.debug()).toMatchSnapshot());

    it('should render divs', () => {
        expect(wrapper.find('div')).toHaveLength(2);
    });
});
