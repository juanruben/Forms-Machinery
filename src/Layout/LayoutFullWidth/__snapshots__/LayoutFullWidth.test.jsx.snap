// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutFullWidth should render correctly 1`] = `
"<div className=\\"full-width-container\\">
  <div className=\\"full-width-container__box\\" />
</div>"
`;
